readme.md

```diff
@@ -8,7 +8,7 @@ For the following quiz, there will be a series of questions or directions follow
 What is Git, what problem does it solve?
 
 ```
-answer goes here
+Git is a set of remote repositories used to track changes to documents/files.
 
```

Comment:
If that's what git is, what is github?

Git is version control software that allows us to code in freedom and collaborate with others. It's a tool that we use that allows us to manage the state of our code at any given time.